draft-ietf-capport-architecture.xml

```diff
@@ -112,24 +112,40 @@
        principles:
       </t>
       <ul spacing="normal">
-        <li>Solutions SHOULD NOT require the forging of responses from DNS or
-            HTTP servers, or any other protocol.  In particular, solutions
-            SHOULD NOT require man-in-the-middle proxy of TLS traffic.</li>
-        <li>Solutions MUST operate at the layer of Internet Protocol (IP) or
-            above, not being specific to any particular access technology such
-            as Cable, WiFi or mobile telecom.</li>
-        <li>Solutions MAY allow a device to be alerted that it is in a
-            captive network when attempting to use any application on the
-            network.</li>
+        <li>Current captive portal solutions typically implement some variations
+            of forging DNS or HTTP responses.
+            Some reportedly perform man-in-the-middle (MITM) proxy of HTTPS in
```

Comment:
```suggestion
            Some attempt man-in-the-middle (MITM) proxy of HTTPS in
```

I've seen plenty of attempts, and I have heard that sometimes this is successful.  But we don't have to say that; attempts are bad enough.

FWIW, my home router (which my ISP insists upon to an insane degree) will throw up a captive portal when the link to the outside world goes down.  That attempts TLS interception.